### DISCUSSION TITLE
PNP web search showing incorrect time

### DISCUSSION THREAD

**Discussion by Neenz13** [microsoft-search/pnp-modern-search#2792] [Q&A]
Hi there

I have a pnp web search on my sharepoint page which connects to one of my document library. I created a managed property for one of my columns that is a date format. Linking this to my pnp search is fine, problem is once linked it shows a completely different date from what is actually showing on the library. Is there a way of getting this to show using an expression?

**Comment by kasperbolarsen** [comment]
Please check the mapping again, and verify the date in this managed property using SharePoint search query tool or similar.
Please share the mapping.

**Reply by Neenz13** [reply]
Hi @kasperbolarsen thank you for your prompt reply. I ended up using the handlebar expression on #115 and that fixed my issue :)
